module1-introduction-to-backend/r3-introduction-to-expressjs/README.md

```diff
@@ -0,0 +1,44 @@
+# Introduction to Express.js
+Express is the most popular Node web framework, and is the underlying library for a number of other popular Node web frameworks. The objectives of this lesson are:
+1. Getting familiar with Express.js
+2. Getting ready to build a backend application using Node and Express
+
+## Why Express?
```

Comment:
i think fromt he perspective of a student, this section needs to explain what the difference between express and node is. in particular the `app.get` stuff is basically the most concrete difference, in that it will make it a lot easier

i also suggest mentioning that rarely you would do only a pure node app for a web server, almost always you'd use some framework